### PR TITLE
add homepage background image

### PR DESCRIPTION
**Pull request by melissarudofsky**

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -20,9 +20,13 @@ import greenPatternImage from '../images/1x/pattern-green.png';
 import meetupsImage from '../images/1x/Meetups.png';
 import buildImage from '../images/1x/Build.png';
 import GradientUnderline from '../components/dev-hub/gradient-underline';
+import BACKGROUND_IMAGE from '../images/1x/homepage-background.png';
```
Comment (jestapinski) [inline src/pages/index.js]: Should we use all caps for these imports as a standard?

Comment (greggb) [reply src/pages/index.js]: why

**Comment by greggb** [comment]
Can we run all the images through https://tinypng.com/ or a related Command line tool. They seem huge

File: src/pages/index.js (hunk 1 above)
Comment (jestapinski) [reply src/pages/index.js]: Just curious if we wanted anything special for images

Comment (melissarudofsky) [reply src/pages/index.js]: made it lower case

**Pull request merged**